python/ray/_private/ray_constants.py

```diff
@@ -534,8 +534,22 @@ def gcs_actor_scheduling_enabled():
 
 RAY_BACKEND_LOG_JSON_ENV_VAR = "RAY_BACKEND_LOG_JSON"
 
+# Write export API event of all resource types to file if enabled.
+# RAY_enable_export_api_write_config will not be considered if
```

Comment:
maybe we should have this flag always be considered. This can be used as a hard shut off if this is not set or set to False.

Reply:
This change wouldn't be backward compatible if `RAY_enable_export_api_write_config` is always considered, right? I was thinking we can eventually deprecate `RAY_enable_export_api_write`, which should have the same effect.